ietf/doc/views_doc.py

```diff
@@ -261,7 +261,9 @@ def document_main(request, name, rev=None, document_html=False):
         iesg_state_summary = doc.friendly_state()
         irsg_state = doc.get_state("draft-stream-irtf")
 
-        can_edit = has_role(request.user, ("Area Director", "Secretariat"))
+        can_edit = has_role(request.user,
+                            ("Secretariat") if iesg_state_slug == 'dead' else
+                            ("Area Director", "Secretariat"))
```

Comment:
The scope of this change is too big. We would need to review each of the many places this would change access for the IESG separately and get the IESG to buy in.

The switch should remain in the template, and should only affect editing the IESG state.